frontend/lib/norent/letter-builder/know-your-rights.tsx

```diff
@@ -56,8 +58,9 @@ export const StateWithProtectionsContent: React.FC<NorentMetadataForUSState> = (
   <>
     <p>{props.lawForBuilder.textOfLegislation}</p>
     <p>
-      We’ve partnered with <PartnerLink {...props.partner} /> to provide
-      additional support once you’ve sent your letter.
+      We’ve partnered with{" "}
+      <PartnerLink {...(props.partner || DefaultStatePartnerForBuilder)} /> to
+      provide additional support once you’ve sent your letter.
```

Comment:
Did we also want to always list the default state partner in addition to the optional one? my understanding from today's chat was that we _always_ list RTTC but also additionally (optionally) list the local state partner if it's present, but maybe i'm mistaken?

Reply:
Oh right @toolness ! I thought that would be addressed in a future PR (once we have the check box for agreeing to let RTTC contact), but I can address it here if you'd like! Lmk what makes your life easiest

Reply:
Oh ok that's cool! merge away!

Reply:
great!